assignments/prototypes.js

```diff
@@ -15,14 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject(p) {
+    this.createdAt = p.createdAt;
+    this.name = p.name;
+    this.demensions = p.dimensions;
```

Comment:
The reason you are getting an undefined on your second console.log below is due to a typo error here.  you  misspelled this.dimensions as this.demensions.  If you used VSCode as your editor, it would have caught this error for you!